File: 1051-height-checker/1051-height-checker.cpp
class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> expected;
        for(int n : heights) {
            expected.push_back(n);
        }
        sort(expected.begin(), expected.end());
        
        int answer = 0;
        for(int i = 0; i < expected.size(); i++){
            if(expected[i] != heights[i]) answer++;
        }
        
        return answer;
    }
};

File: 1051-height-checker/1051-height-checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int N = heights.length;
        int[] ori = new int[N];
        for(int i = 0; i < N; i++) {
            ori[i] = heights[i];
        } 
        
        Arrays.sort(heights);
        
        int answer = 0;
        for(int i = 0; i < N; i++) {
            if(ori[i] != heights[i]) answer++;
        }
        
        return answer;
    }
}